pkg/api/customization/monitor/cluster_graph_action.go

```diff
@@ -63,6 +68,12 @@ func (h *ClusterGraphHandler) QuerySeriesAction(actionName string, action *types
 		return fmt.Errorf("get usercontext failed, %v", err)
 	}
 
+	mgmtClient := h.clustermanager.ScaledContext.Management
+	nodeLister := mgmtClient.Nodes(metav1.NamespaceAll).Controller().Lister()
```

Comment:
never get a lister in business logic like this. you have to get the lister during "setup" of rancher.